charts/prometheus-couchdb-exporter/README.md

```diff
@@ -1,80 +1,68 @@
 # prometheus-couchdb-exporter
 
-[couchdb-prometheus-exporter](https://github.com/gesellix/couchdb-prometheus-exporter) is a Prometheus exporter for CouchDB metrics.
+A Prometheus exporter for [CouchDB](https://couchdb.apache.org/) metrics.
 
-## TL;DR;
+This chart bootstraps a [CouchDB Exporter](https://github.com/gesellix/couchdb-prometheus-exporter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-```bash
-$ helm install stable/prometheus-couchdb-exporter
+## Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
+
+## Get Repo Info
+
+```console
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```

Comment:
@scottrigby do you still want to have the stable here?

Reply:
Good catch. No we only want this on the prometheus chart, since only it and the deprecated still older named `prometheus-operator` chart (see #9) have dependencies in the stable repo.